backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -19,46 +19,48 @@
 @Service
 public class UserServiceImpl implements UserService {
 
-  @Autowired private UserRepository userRepository;
-  @Autowired private RoleRepository roleRepository;
-  @Autowired private BCryptPasswordEncoder bCryptPasswordEncoder;
-  @Autowired private ModelMapper modelMapper;
+    @Autowired private UserRepository userRepository;
```

Comment:
you already used the approach with Lombok in CustomAuthenticationFilter.class. Use the same here instead of Autowired